codeforamerica/summer-ebt-surveyjs-demo
# feat: add mixpanel page load events

- Integrate mixpanel into the demo
- Support environment variables by creating a template which should be locally copied and filled out for each environment
- Track a mixpanel event called page_load when a new page within the survey is loaded, which can be used to track progress within the form
- Allow disabling Mixpanel entirely

Steps to test
- Follow instructions in environment.template to copy the file
- Add the Mixpanel access token in the dev environment file and set useMixpanel to true
- Build and deploy the application as usual, and open it locally
- Navigate through a few different pages of the survey
- In Mixpanel, open the "SurveyJS Summer EBT demo" project
- Create a report tracking the page_load metric with a breakdown by page_name. This should show the pages you navigated through in the survey, showing that the application is connected to Mixpanel

You can also test setting useMixpanel to false. In that case you will see that any navigation through the form does not result in Mixpanel events.

More mixpanel event types and more detailed tracking data will be added in a future PR
